### PR TITLE
WIP: posting data from client side

### PR DESCRIPTION
**Pull request by mallett002**
I'm trying to perform all CRUD operations using JS instead of html. 
The html would look like:
``` html
<form method="post" action="/api/issues">
```
How do I get the post data, or the `req.body` when I want to do it on the client side in JS?
I'm using `axios`, an HTTP client to send the post request. Do I even need to send a post body, or does it do it automatically? The examples in the axios [docs](https://www.npmjs.com/package/axios#axiosposturl-data-config) don't show how to get the actual data.

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+       .catch(e => {
+          throw Error(e);
+       });
+
+   //    // Trying to figure out how to insert post data from the inputs, Or do I even have to?
+   // axios.post("/api/issues/", {
+   //    // data from the inputs
+   //    title: input.name ?
+   // })
+});
```
Comment (mallett002) [inline client/index.js]: Also, the post isn't making it into the controller for that route.

**Comment by yoans** [comment]
You are doing a few too many things at once here.
Your might be having issues because server.js not running as a part of your npm start.
Change your server port to 8080 and run `node server.js`.
Otherwise, it looks like you are on the right track.

**Comment by yoans** [comment]
I suppose it might just be the port number.

**Comment by mallett002** [comment]
Figured it out. The methods `put`, `post`, and `delete` do need a payload to send to the server.

**Pull request merged**